src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -101,6 +101,8 @@ public static class WebScriptHostBuilderExtension
                     // Hosted services
                     services.TryAddEnumerable(ServiceDescriptor.Singleton<IHostedService, HttpInitializationService>());
                     services.TryAddEnumerable(ServiceDescriptor.Singleton<IHostedService, FileMonitoringService>());
+
+                    ConfigureRegisteredBuilders(services, rootServiceProvider);
```

Comment:
Only product change. Allows us to register a builder for any services -- and it will be called last to allow us to override.

Reply:
Looking back on this now -- this seems confusing. We have a way to configure WebJobsBuilder and a way to configure IServiceCollection (which happens later). @fabiocav -- do you remember if there was a reason that all of these services are set up in ConfigureServices here and not in the IWebJobsBuilder passed to AddScriptHost above?